tensorflow/lite/kernels/internal/common.h

```diff
@@ -565,7 +565,8 @@ log_x_for_x_greater_than_or_equal_to_1_impl(
   // required shift "ourselves" instead of using, say, Rescale.
   FixedPoint0 z_a = FixedPoint0::FromRaw(input_val.raw());
   // z_a_pow_2 = input_integer_bits - z_a_headroom;
-  int z_a_headroom_plus_1 = CountLeadingZeros(static_cast<uint32_t>(z_a.raw()));
+  int32_t z_a_headroom_plus_1 =
```

Comment:
Is this type change required for correctness, or is it just a cleanup step? If it's just cleanup, I'd prefer to avoid the code churn.

Reply:
This was required to compile on ARM platforms.  However, the fix was incorrect as it had int and int32_t being used interchangeably without a static_cast.

Fixed.